CRM/Upgrade/Incremental/sql/4.7.beta1.mysql.tpl

```diff
@@ -14,7 +14,9 @@ INSERT INTO civicrm_navigation
 VALUES
 ( {$domainID}, 'civicrm/a/#/status', '{ts escape="sql" skip="true"}System Status{/ts}', 'System Status', 'administer CiviCRM', '', @parent_id, '1', NULL, 0 );
 
+-- CRM-17147 People with empty deceased-flag ('is null') get removed from recipient list of a mailing
 UPDATE civicrm_contact SET is_deceased = 0 WHERE is_deceased IS NULL;
+ALTER TABLE civicrm_contact ADD DEFAULT 0 FOR is_deceased;
```

Comment:
I think there was some fault in my code. This should hopefully pass after the changes.